gradle.properties

```diff
@@ -1,4 +1,4 @@
-release_version=4.2.0
+release_version=4.2.1
```

Comment:
I believe we must update the minor version because we update minor versions in dependencies

Reply:
done